zpl2.py

```diff
@@ -35,6 +35,32 @@
 ARG_COLOR = 'color'
 ARG_ROUNDING = 'rounding'
 ARG_DIAMETER = 'diameter'
+ARG_QRMODEL = 'qrmodel'
+ARG_QRMAGNIFICATION = 'qrmagnification'
+ARG_QRERRORCORRECTION = 'qrerrorcorrection'
+ARG_QRMASK = 'qrmask'
+
+# QR Code Model values
+QRMODEL_ORIGINAL = 1
+QRMODEL_ENHANCED = 2
+
+# QR Code Magnification Factor
+QRMAGNIFICATION_150DPI = 1
+QRMAGNIFICATION_200DPI = 2
+QRMAGNIFICATION_300DPI = 3
+QRMAGNIFICATION_400DPI = 4
+QRMAGNIFICATION_500DPI = 5
+QRMAGNIFICATION_600DPI = 6
+QRMAGNIFICATION_700DPI = 7
+QRMAGNIFICATION_800DPI = 8
+QRMAGNIFICATION_900DPI = 9
+QRMAGNIFICATION_1000DPI = 10
+
+# QR Code Error Correction
+QRERRORCORRECTION_ULTRA_HIGH = 'H'
```

Comment:
Please rename these constants to `ERROR_CORRECTION_*` to follow the naming of the official ZPL2 documentation.